packages/terra-functional-testing/src/commands/expect/toMatchReference.js

```diff
@@ -38,7 +38,7 @@ function toMatchReference(screenshot) {
 
   let pass = global.Terra.serviceOptions.ignoreScreenshotMismatch || imagesMatch;
 
-  if (global.Terra.serviceOptions.useRemoteReferenceScreenshots && global.Terra.serviceOptions.buildBranch === BUILD_BRANCH.pullRequest && !pass) {
+  if (global.Terra.serviceOptions.useRemoteReferenceScreenshots && global.Terra.serviceOptions.buildBranch.includes(BUILD_BRANCH.pullRequest) && !pass) {
```

Comment:
Is there any way we can be more strict in checking for the branch name? If I create a feature branch that starts the "pr" (for examples: `project-orion-upgrade`, `pretty-format-report`, `provider-search-fix`), any of these feature branch build would satisfy this check when it shouldn't.

Could we possibly check for the `-` after `pr` followed by a number?

Reply:
swapped to Regex expression here https://github.com/cerner/terra-toolkit/pull/737/commits/8564a3d40cbc535ac0198ee1a905ad0b22d65cfb#diff-d74c0126120632caa0af1cd9e75517e7f5b64d84836d4301f4d530cfb5256596R31